cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -269,7 +269,7 @@
 
     // The "search" button needs to be more flexible than in our usual
     // input-w-btn element to accommodate longer translations of "search",
-    // so we'll make it flexbox
+    // so we will make it flexbox
```

Comment:
This got fixed in https://github.com/cfpb/consumerfinance.gov/pull/7274 but it looks like there's no conflict so no need to change anything.

Reply:
Mmmm I guess I didn't pull. I was thinking it was a different one.